Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -613,7 +613,11 @@ void NetPlayDialog::UpdateGUI()
 
   static const std::map<NetPlay::SyncIdentifierComparison, QString> player_status{
       {NetPlay::SyncIdentifierComparison::SameGame, tr("OK")},
-      {NetPlay::SyncIdentifierComparison::DifferentVersion, tr("Wrong Version")},
+      {NetPlay::SyncIdentifierComparison::DifferentHash, tr("Game file has a different hash")},
```

Comment:
This is technically misleading, as it implies the hash of the file itself is different rather than the sync hash, but I'll let that slide since the concept of a sync hash isn't something we can succinctly explain to users. And it is true that if the sync hash is different, the file hash is different – just not vice versa.

Reply:
I think this is good enough, since as you say if the sync hash is different, the file hash is different.  Perhaps we should direct the user to the verify tab in this case, though?  (Assuming there is enough space on the UI)

Reply:
There is very little space in the UI – you're already stretching it. So I'd say let's leave this as it is.